static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -71,32 +126,49 @@ export function InviteBanner({missingMembers, onSendInvite, organization}: Props
   cards.push(<SeeMoreCard key="see-more" missingUsers={users} />);
 
   return (
-    <StyledCard data-test-id="invite-banner">
-      <CardTitleContainer>
-        <CardTitleContent>
-          <CardTitle>{t('Bring your full GitHub team on board in Sentry')}</CardTitle>
-          <Subtitle>
-            {tct('[missingMemberCount] missing members that are active in your GitHub', {
-              missingMemberCount: users.length,
-            })}
-            <Tooltip title="Based on the last 30 days of commit data">
-              <IconInfo size="xs" />
-            </Tooltip>
-          </Subtitle>
-        </CardTitleContent>
-        <ButtonContainer>
-          <Button
-            priority="primary"
-            size="xs"
-            // TODO(cathy): open up invite modal
-            data-test-id="view-all-missing-members"
-          >
-            {t('View All')}
-          </Button>
-        </ButtonContainer>
-      </CardTitleContainer>
-      <MemberCardsContainer>{cards}</MemberCardsContainer>
-    </StyledCard>
+    <Feature organization={organization} features={['integrations-gh-invite']}>
+      <StyledCard data-test-id="invite-banner">
```

Comment:
Do you need the test ID, can you instead use the heading text and do something like 

```tsx
getByRole('heading', {name: 'Bring your full GitHub team on board in Sentry'})
```

That would be a much more semantic way to approach this

Reply:
heading doesn't work since this text is inside a styled div, but getByText/queryByText does

Reply:
Maybe it should be a heading though yeah? Looks like a heading I think!